examples/native-client/build.gradle.kts

```diff
@@ -1,12 +1,12 @@
 plugins {
     application
     kotlin("jvm")
-    id("com.palantir.graal") version "0.7.2"
+    id("com.palantir.graal") version "0.10.0"
 }
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
```

Comment:
I'm not an expert on this, but wouldn't we want to have `targetCompatibility` be the lowest version of Java we want to support (e.g. `VERSION_1_8`)?

Reply:
This one is for the native-client which uses GraalVM and there is no longer JDK 1.8 support. So for this part, 11 is the minimum.